src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
Yeah I've thought about it, but wouldn't there be too many items in it? If a site is blocked afterwards, Lantern will not notice it.